docs/source/library/Architecture.rst

```diff
@@ -0,0 +1,52 @@
+Architecture
+============
+
+The *architecture* describes the properties of the target device when mapping a quantum circuit.
```

Comment:
It would be helpful to mentioned the different possibilities for constructing architectures here (similar to the README).

Most importantly, the first section following this introduction should be the import of existing Qiskit architectures, since this is the recommended way of working with QMAP. 

From the readme assuming `arch` is a Qiskit backend:
```python3
from mqt.qmap.qiskit.backend import import_backend
    architecture = import_backend(arch)
```

I'd probably end this introductory part with a list of possible ways to construct an architecture and then dedicate a (short) subsection to each of those.

Reply:
Now there should be a comprehensive tutorial on how to construct architectures.